proxy_test.go

```diff
@@ -74,7 +75,9 @@ func TestGetViaProxy(t *testing.T) {
 	requests := make(chan string, 2)
 	server := httptest.NewServer(testServer{requests})
 	defer server.Close()
-	proxy := httptest.NewServer(testProxy{requests, "proxy", newDirectProxy()})
+	// Proxy request should not go to the mux. The empty mux will always return 404
```

Comment:
Nit: need a period after the end of the sentence.

Reply:
Done. Fixed in e8a57324da094f3e748dc08e24f188b127c4fd5a